services/director-v2/src/simcore_service_director_v2/modules/docker_daemon.py

```diff
@@ -0,0 +1,51 @@
+""" Module responsible of interacting with docker daemon
+
+
+"""
+# TODO: code below simply copied from old director
+import logging
+
+import aiodocker
+from asyncio_extras import async_contextmanager  # Lt's try to avoid this??
```

Comment:
will this remain like this or something else should replace it?

Reply:
as soon as we move to python >=3.7 we can remove this